app/clparamiko.py

```diff
@@ -0,0 +1,40 @@
+import paramiko
+import time
+from contextlib import contextmanager
+
+from args import Arguments
+from shell import Shell
+
+class ClParamiko(Arguments, Shell):
+    def __init__(self, args, password):
+        Arguments.__init__(self, args, password)
+
+    @contextmanager
+    def connect(self):
+        con = paramiko.SSHClient()
+        con.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        con.connect(hostname=self.host, port=self.port, username=self.user, password=self.password)
+        yield con
+        con.close()
+
+    def response(self, channel):
+        while not channel.recv_ready():
+            time.sleep(0.1)
+        stdout = ''
+        while channel.recv_ready():
+            stdout += channel.recv(1024)
+        return stdout
+
+    def send_connect(self):
```

Comment:
@GitSergeyGit the usage contract is poorly designed: 
Imagine you're using this API. You're creating a `Shell` and then you have 3 public methods that initiates execution:
* run
* send_connect
* connect
* response

We need to have obvious contract that raises no question on how to use it. Lets close all non public methods under single underscore, like `_method_name`